Dockerfile-alpine.template

```diff
@@ -1,6 +1,7 @@
 FROM alpine:0.0
 
 ENV NODE_VERSION 0.0.0
+ENV PYTHON_VERSION python3
```

Comment:
I think you can probably do it without the explicit ENV here. I think that's how the CHECKSUM one is done